components/Gameboard.tsx

```diff
@@ -7,7 +7,7 @@ import { checkWinner } from "../utilities/checkWinner";
 
 export const Gameboard: FunctionComponent = ({}) => {
   const [currentPlayer, setCurrentPlayer] = useState(true);
-  const [currentBoard, setCurrentBoard] = useState([0, 0]);
+  const [currentBoard, setCurrentBoard] = useState<number[] | null>(null);
```

Comment:
Also `[number, number]` wär da schon der schönere type. Oder noch einen extra type für 0-2 erstellen und den statt `number` verwenden!

Reply:
Man kann es auch übertreiben :joy: 
So viel Zeit habe ich dann doch nicht...
Schlage mich lieber mit so einem Kram rum: https://stackoverflow.com/questions/13291578/how-to-localize-an-android-app-in-indonesian-language